filters/auth/forwardtokenpart_test.go

```diff
@@ -0,0 +1,147 @@
+package auth
+
+import (
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"net/http/httptest"
+	"net/url"
+	"testing"
+	"time"
+
+	"github.com/zalando/skipper/eskip"
+	"github.com/zalando/skipper/filters"
+	"github.com/zalando/skipper/net"
+	"github.com/zalando/skipper/proxy/proxytest"
+)
+
+func TestForwardTokenPart(t *testing.T) {
```

Comment:
Maybe we do not need full-fledged integration test that is hard to follow for a filter that simply gets and puts value from the state bag. 

Please have a look at e.g. https://github.com/zalando/skipper/blob/master/filters/ratelimit/backendratelimit_test.go and may be you would decide to rewrite it into a unit test.